jest.config.mjs

```diff
@@ -60,6 +60,7 @@ const config = {
  */
 export default {
   collectCoverageFrom: ['./packages/govuk-frontend/src/**/*.{js,mjs}'],
+  coverageProvider: 'v8',
```

Comment:
**question** Does that affect the coverage of the tests that do run through Babel? I think what I'm worried about is the Babel helpers being considered for the coverage if it is collected via `v8`, but excluded with default setting. I wouldn't block the PR as we're not looking much at coverage at the moment, but it'll be something to dig into if/when we get to https://github.com/alphagov/govuk-frontend/issues/3101 😊

Reply:
Yeah a little bit

Coverage from [`coverageProvider: 'v8'`](https://v8.dev/blog/javascript-code-coverage) is generated from transpiled code but it's mapped back to the original source using source maps and will exclude sneaky helpers. [Read about when it launched in Jest v25](https://jestjs.io/blog/2020/01/21/jest-25#v8-code-coverage)

I'm sure we'll see minor differences in coverage percentages. Turning off Babel transforms for `*.js` stops [`babel-plugin-istanbul`](https://github.com/istanbuljs/babel-plugin-istanbul) from instrumenting the code so it's better than losing it entirely?

Reply:
The situation doesn't sound too bad if the `v8` thing takes sourcemaps into account. Like I said, not a blocker, just wanted to understand a bit and flag for #3101 😊